src/plugins/intel_gpu/tests/fusions/resample_fusion_test.cpp

```diff
@@ -308,3 +309,82 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, resample_scale_concat, ::testing::ValuesIn
     resample_test_params{ CASE_RESAMPLE_U8_3, 3, 6 },
     resample_test_params{ CASE_RESAMPLE_U8_4, 3, 6 },
 }));
+
+class resample_scale_fusing_through : public ResamplePrimitiveFusingTest {};
+TEST_P(resample_scale_fusing_through, reshape) {
+    auto p = GetParam();
+    auto reshape_shape = p.out_shape;
+    reshape_shape.feature[0] *= reshape_shape.spatial[0];
+    reshape_shape.spatial[0] = 1;
+
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("scale_data", get_mem(layout{ p.default_type, p.default_format, tensor{ 1, 1, 1, 1 } })),
+        resample("resample_prim", "input", p.out_shape, p.in_shape.feature[0], p.type),
+        reshape("reshape", "resample_prim", reshape_shape),
+        eltwise("scale", "reshape", "scale_data", eltwise_mode::prod),
+        reorder("reorder_bfyx", "scale", p.default_format, data_types::f32)
+    );
+
+    tolerance = 1e-5f;
```

Comment:
I think default checks don't guarantee that eltwise is fused to resample node, i.e. it can be fused to reshape which is not expected. Maybe it makes sense to extend test checks somehow to ensure that the fusion pass does what we really wanted.

Reply:
I have added `check_fusions_correctness(network&, std::map<std::string, std::vector<std::string>>&)` method to the `BaseFusingTest` class which takes std::map with expected fusions